nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -86,17 +91,130 @@ Returns a faction datum by its name (case insensitive!)
 		if(F.id == id)
 			return F
 
-
 /datum/controller/subsystem/star_system/proc/add_blacklist(what)
 	enemy_blacklist += what
 	if(locate(what) in enemy_types)
 		enemy_types -= what
 
-/datum/controller/subsystem/star_system/proc/instantiate_systems()
+/datum/controller/subsystem/star_system/proc/instantiate_systems(_source_path = SSmapping.config.starmap_path)
+	message_admins("Loading starsystem from [_source_path]...")
+	var/list/_systems = list()
+	//Read the file in...
+	try
+	{
+		_systems = json_decode(rustg_file_read(file(_source_path)))
+	}
+
+	catch(var/exception/ex){
+		//Fallback: Load the hardcoded systems and report an error.
+		instantiate_systems_backup()
+		CRASH("FATAL: Unable to load starmap from: [_source_path]. (Defaulting...): [ex]")
+	}
```

Comment:
kmc no 😭😭😭

Reply:
~~fixed~~ made even worse in the latest commit :)